Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM library/ubuntu:artful
+
+# Create app directory
+WORKDIR /usr/src/site
+
+# Install app dependencies
+COPY . .
+
+# Bootstrap website
+RUN apt-get update && \
```

Comment:
Is there a reason these are all one giant RUN command? I would be separating them out (so Docker can cache individual steps if we need to change something). Furthermore, combine the apt operations into one step so we only have to interact with the pkg manager once.

Reply:
I hacked this up while sleep deprived, so I *think* the reason I mushed it all into one `RUN` command was because each `RUN` command is run on a different ephemeral container. An update to the APT cache on one wouldn't affect the next container's APT cache.